browser/src/map/handler/Map.Keyboard.js

```diff
@@ -436,6 +436,37 @@ L.Map.Keyboard = L.Handler.extend({
 				this._map._docLayer._preview.partsFocused = false;
 			}
 		}
+		else if (this._isCtrlKey(ev) && !ev.shiftKey && !ev.altKey && ev.keyCode === this.keyCodes.F) {
```

Comment:
this is example of command we should put into KeyboardShortcuts.ts (in separate commit):
it has easy way to define language specific shortcuts: https://github.com/CollaboraOnline/online/blob/master/browser/src/map/handler/Map.KeyboardShortcuts.ts#L192